app/elements/common/services/challenge-service.html

```diff
@@ -13,26 +13,7 @@
 			handle-as="json"
 			content-type="application/json"
 			last-response="{{challenge}}"
-			with-credentials="true"
-			on-response="_onResponse"
-			on-error="_onError">
-		</iron-ajax>
-		<iron-ajax
-			id="getSingleChallenge"
-			method="GET"
-			handle-as="json"
-			content-type="application/json"
-			last-response="{{challenge}}"
-			with-credentials="true"
-			on-response="_onResponse"
-			on-error="_onError">
-		</iron-ajax>
-		<iron-ajax
-			id="getMultiChallenges"
-			method="GET"
-			handle-as="json"
-			content-type="application/json"
-			last-response="{{challenges}}"
+			body="{{challenge}}"
```

Comment:
set body in save function..it`s too brainfucking to have body = challenge and lastresponse = challenge